libs/utils/utils.js

```diff
@@ -654,6 +654,16 @@ export function loadPrivacy() {
   });
 }
 
+async function loadJarvisChat() {
+  const config = getConfig();
+  const jarvis = getMetadata('jarvis-chat');
+  if (!config.jarvis?.id || !config.jarvis?.version) return;
+  if (jarvis === 'on') {
```

Comment:
Mind giving a bit of background on how milo features work? Does a consumer now explicitly need to state the feature is on, or is it on by default?

Reply:
The feature is disabled by default, it's only needed for certain projects. Milo consumers will have to provide a chat configuration and turn the feature on on specific pages (or all) via metadata.